index.html

```diff
@@ -40,7 +40,17 @@
         company: "Microsoft Corporation",
         companyURL: "https://microsoft.com/",
         w3cid: 60712
-      },],
+      }, {
+        name: "Kenneth Rohde Christiansen",
+        company: "Intel Corporation",
+        companyURL: "https://intel.com/",
+        w3cid: 57705
+      }, {
```

Comment:
Did Kenneth duplicae? :)

Reply:
LOL, Kenneth puts in so much work we need to list him twice!